sfera-mock/src/main/resources/static_sfera_resources/jp/SFERA_JP_500.xml

```diff
@@ -0,0 +1,141 @@
+<?xml version="1.0"?>
+<JourneyProfile JP_Status="Valid" JP_Version="1"
+                xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                xsi:noNamespaceSchemaLocation="../../SFERA_3.0_custom.xsd">
+    <TrainIdentification>
+        <OTN_ID>
+            <Company>1085</Company>
+            <OperationalTrainNumber>500</OperationalTrainNumber>
+            <StartDate>2022-01-04</StartDate>
+        </OTN_ID>
+    </TrainIdentification>
+    <SegmentProfileList SP_ID="500_1" SP_VersionMajor="1" SP_VersionMinor="1" SP_Direction="Nominal">
+        <SP_Zone>
+            <IM_ID>0085</IM_ID>
+        </SP_Zone>
+        <TimingPointConstraints TP_StopSkipPass="Stopping_Point" TP_Information="None">
+            <TimingPointReference>
+                <TP_ID_Reference TP_ID="Genève-Aéroport"/>
+            </TimingPointReference>
+        </TimingPointConstraints>
+        <TimingPointConstraints TP_StopSkipPass="Stopping_Point" TP_Information="None">
+            <TimingPointReference>
+                <TP_ID_Reference TP_ID="Genève"/>
+            </TimingPointReference>
+        </TimingPointConstraints>
+        <TimingPointConstraints TP_StopSkipPass="Passing_Point" TP_Information="None">
+            <TimingPointReference>
+                <TP_ID_Reference TP_ID="Gland"/>
+            </TimingPointReference>
+        </TimingPointConstraints>
+        <TimingPointConstraints TP_StopSkipPass="Passing_Point" TP_Information="None">
+            <TimingPointReference>
+                <TP_ID_Reference TP_ID="Gilly-Bursinel"/>
+            </TimingPointReference>
+        </TimingPointConstraints>
+        <TimingPointConstraints TP_StopSkipPass="Stopping_Point" TP_Information="None">
+            <TimingPointReference>
+                <TP_ID_Reference TP_ID="Morges"/>
+            </TimingPointReference>
+        </TimingPointConstraints>
+        <TemporaryConstraints startEndQualifier="Starts" temporaryConstraintType="ASR" startLocation="700">
```

Comment:
hast du Start- und Endzeit absichtlich weggelassen?

Reply:
ja im 513 ist die zeit vorhanden, so sind beides cases abgedeckt. Lokal gibts noch weiter Testecases wo das Datum in der vergangenheit / zukunft liegt, für den mock server aber nicht interessant